src/gmpints.c

```diff
@@ -568,6 +568,114 @@ Obj ObjInt_Int8( Int8 i )
 #endif
 }
 
+/**************************************************************************
+**
+** The following functions a GAP integer into an Int, UInt, Int8 or UInt8
+** if it is in range, even if it is too big to be a small integer
```

Comment:
Not sure why "small integer" is even mentioned here?

Also, state what happens if it is *not* in range?